spec/models/conference_spec.rb

```diff
@@ -1585,11 +1585,11 @@
   end
 
   describe 'after_create' do
-    let!(:conference) { Conference.new(title: 'ABC',short_title: 'XYZ',start_date: Date.today, end_date: Date.today + 10) }
-    let!(:free_ticket) { conference.tickets.first }
+    before { @conference = Conference.new }
```

Comment:
I actually like the previous version of this test better.
In the original version, we create a conference and test that, once created, it has at least one free ticket.

The new version checks the *implementation*. Meaning: when we create a conference, make sure that a certain method is called.
But what if we break that method and it DOESN'T create a free ticket anymore? This test will still pass, but the conference will not have a free ticket anymore.